run_sdk_container

```diff
@@ -129,7 +129,8 @@ if [ -z "$stat" ] ; then
        -e SDK_GROUP_ID="$(id -g)" \
        --name="$name" \
        --hostname="$hostname" \
-       "${container_image_name}"
+       --entrypoint /bin/sleep \
+       "${container_image_name}" 10
```

Comment:
Won't this circumvent the user id / group id change now? That's done by passing env variables to `sdk_entry.sh` which is only done at container creation time; however we change the entrypoint here to sleep, so `sdk_entry.sh` is never called with the `SDK_USER_ID` and `SDK_GROUP_ID` variables set?

Reply:
(I would strongly prefer moving the sleep change to a separate PR)